src/components/shared/modal/modalContent/modalContent.scss

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+@import 'src/styles/mixins';
+@import 'src/styles/themes';
+
+.modal-content {
+  border-style: solid;
+  width: 400px;
+  height: 200px;
+  z-index: 1;
+  @include rem-fallback(border-width, 1px);
+  @include rem-fallback(padding, 20px);
+
+  @include themify($themes) {
+    color: themed('colorContentText');
+    background-color: themed('colorFlyoutBackground');
+    border-color: themed('colorSelectionBg');
```

Comment:
Mixing color names from the theme (content, flyout, selection). Instead, create named colors for modals specifically.

Reply:
done